include/EclipseMonitor/Trie/ExtensionNode.hpp

```diff
@@ -0,0 +1,86 @@
+// Copyright 2022 Tuan Tran
+// Use of this source code is governed by an MIT-style
+// license that can be found in the LICENSE file or at
+// https://opensource.org/licenses/MIT.
+
+#pragma once
+
+#include "EclipseMonitor/EthKeccak256.hpp"
+#include "EclipseMonitor/Trie/Nibbles.hpp"
+#include "EclipseMonitor/Trie/TrieNode.hpp"
+
+using namespace SimpleObjects;
+
+namespace EclipseMonitor
+{
+
+namespace Extension
```

Comment:
What is the reason for having different namespaces in different headers? it seems redundant.